docs/guides/customization.md

```diff
@@ -0,0 +1,109 @@
+# Defining setup/teardown tasks, context, and `nnbench.Parameters`
+
+This page introduces some customization options for benchmarks runs.
+These options can be helpful for tasks surrounding benchmark state management, such as automatic setup and cleanup, contextualizing results with context values, and defining typed parameters with the `nnbench.Parameters` class.
+
+## Defining setup and teardown tasks
+
+For some benchmarks, it is important to set certain configuration values and prepare the execution environment before running.
+To do this, you can pass a setup task to all of the nnbench decorators via the `setUp` keyword:
+
+```python
+import os
+
+import nnbench
+
+
+def set_envvar(**params):
```

Comment:
I would explicitly state that the `setUp` and `tearDown` functions must take all members of `params` as input. Otherwise it might not be clear why we pass `**params`. 

On that note, this is only stated in the `Benchmark` class docstring, not in the decorators. Should we add that there as well?

Reply:
Yes.

Reply:
Added in [d56cac4](https://github.com/aai-institute/nnbench/pull/57/commits/d56cac46fb4729b991be0c96bd8e148ae3a36731), I will defer the decorator mention to a follow-up.